src/packages/solid/components/Icon/Icon.tsx

```diff
@@ -51,10 +51,12 @@ const Icon: Component<IconProps> = props => {
       {...props}
       tone={props.tone ?? styles.tone}
       style={[
+        ...props.style,
```

Comment:
we should only spread this if its actually an array https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray